nengo_gui/config.py

```diff
@@ -47,10 +47,11 @@ def dumps(self, uids):
                 lines.append('%s = %s' % (uid, obj.code_python(uids)))
                 for k in obj.config_defaults.keys():
                     v = getattr(self[obj], k)
-                    if isinstance(v, bool):
-                        val = '%s' % v
-                    else:
+                    try:
                         val = '%g' % v
```

Comment:
If ```v``` is True, this will result in the string ```'1'```, rather than ```True```.  That's why the isinstance check was there.